lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -752,7 +756,8 @@ export class BrowserCacheManager extends CacheManager {
 export const DEFAULT_BROWSER_CACHE_MANAGER = (clientId: string, logger: Logger) => {
     const cacheOptions = {
         cacheLocation: BrowserCacheLocation.MemoryStorage,
-        storeAuthStateInCookie: false
+        storeAuthStateInCookie: false,
+        secureCookies: false
```

Comment:
Should we separate cookie config into its own object?

Reply:
I'd be interested in hearing others' opinions. If we foresee adding more cookie configuration in the future, then I'd definitely vote we make a dedicated object under CacheConfig.

Reply:
I'd say in v3 lets do that, will be a bit weird given we wont be able to put storeAuthStateInCookie in it now.

Reply:
sounds good.

Reply:
Agree to make this change this in a major version.